lib/chef/provider/package/zypper.rb

```diff
@@ -145,8 +145,10 @@ def zypper_package(command, global_options, *options, names, versions)
           zipped_names = zip(names, versions)
           if zypper_version < 1.0
             shell_out!("zypper", global_options, gpg_checks, command, *options, "-y", names)
+            sleep 120
           else
             shell_out!("zypper", global_options, "--non-interactive", gpg_checks, command, *options, zipped_names)
+            sleep 120
```

Comment:
is this still just for testing?  that's a long time...

Reply:
zypper process is locked by the previous test to we have seen it fail multiple times in the release builds, the sleep time allows the process to complete and unlock before the next verification step triggers

Reply:
https://github.com/chef/chef/pull/8742 . in progress, good to merge once this passes

Reply:
@lamont-granquist found a correct fix for zypper issue we are seeing on the SLES tester, will add that add it in a different PR.